PCAxis.Charts/ChartHelper.cs

```diff
@@ -145,6 +145,8 @@ public static PxWebChart GetChart(ChartSettings settings, PXModel model)
             chart.Palette = ChartColorPalette.None;
             chart.PaletteCustomColors = lstColors.ToArray();
 
+            Color backgroundColor = ColorTranslator.FromHtml(settings.BackgroundColor);
+            chart.BackColor = Color.FromArgb(settings.BackgroundAlpha, backgroundColor);
```

Comment:
According to https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.datavisualization.charting.chart.backcolor?view=netframework-4.8#remarks transparent backgrounds are not supported. But we are able to set it this although this messes up the AntiAlising of the texts in the chart, this is probably due to the information given in this post https://social.msdn.microsoft.com/Forums/en-US/64c7d890-c98b-4392-9ec9-1b4af3e24300/labels-bold-when-background-transparent?forum=MSWinWebChart

We can make it lokk better by adding the following code after the backcolor assigment
```
if (settings.BackgroundAlpha != 255)
{
   chart.AntiAliasing = AntiAliasingStyles.Graphics;
}
```

Reply:
The suggested fix does indeed make the text look much better!
![antialiasingchange](https://user-images.githubusercontent.com/21987439/182384678-c8ae8c2f-0ace-49e9-bbde-a110d2ebdd9e.png)
I have commited the new fix. acd643d89f02e43c329a1b1325413d955ed635cd